src/scaffoldmaker/meshtypes/meshtype_3d_stellate1.py

```diff
@@ -470,6 +539,7 @@ def generateBaseMesh(cls, region, options):
                 element = mesh.findElementByIdentifier(elementID)
                 markerLocation.assignMeshLocation(cache, element, addMarker["xi"])
 
+        fm.endChange()
```

Comment:
Remove beginChange/endChange as this is handled by Scaffold_base generateMesh().
I couldn't see beginChange? These must always be used in pairs, but note we don't use them any more in scaffoldmaker, instead we use the ChangeManager(fieldmodule) from opencmiss utils, which automatically calls endChange when we leave the block of code.